src/ir_Sharp.h

```diff
@@ -122,8 +122,16 @@ const uint8_t kSharpAcTimerHoursMax =                             0b1100;  // 12
 const uint8_t kSharpAcOffTimerType =                           0b0;
 const uint8_t kSharpAcOnTimerType =                            0b1;
 
-const uint8_t kSharpAcSwingToggle =                0b111;
-const uint8_t kSharpAcSwingNoToggle =              0b000;
+// Ref: https://github.com/crankyoldgit/IRremoteESP8266/discussions/1590#discussioncomment-1260213
+const uint8_t kSharpAcSwingVOff =    0b000;  // Stop swinging.
+const uint8_t kSharpAcSwingVHigh =   0b001;  // 0° down aka Coanda (Cool)
+const uint8_t kSharpAcSwingVOff1 =   0b010;  // Off, but go to last fixed pos.
+const uint8_t kSharpAcSwingVMid =    0b011;  // 30° down
+const uint8_t kSharpAcSwingVLow =    0b100;  // 45° down
+const uint8_t kSharpAcSwingVOff2 =   0b101;  // Off, but go to last fixed pos.
+const uint8_t kSharpAcSwingVCoanda = 0b110;  // 0° down (Cool), 75° down (Heat)
+const uint8_t kSharpAcSwingVLowest = kSharpAcSwingVCoanda;
+const uint8_t kSharpAcSwingVAuto =   0b111;  // Constant swinging
```

Comment:
I still understood this as being a toggle

Reply:
And so it is. It's back to a "toggle" of sorts.